packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/optimize-ciemss-operation.ts

```diff
@@ -163,15 +163,26 @@ export const OptimizeCiemssOperation: Operation = {
 // Get the optimization result file
 // Concat the optimization result file with the optimization interventions from simulation object
 export async function getOptimizedInterventions(optimizeRunId: string) {
+	// This is a camel case version of types/Types OptimizeInterventions as we are reading from simulation execution payload
+	interface OptimizeInterventionsCamelCase {
+		intervention_type: string;
+		param_names: string[];
+		param_values?: number[];
+		start_time?: number[];
+		objective_function_option: string;
+		initial_guess?: number[];
+		relative_importance: number;
+	}
```

Comment:
Your comment says camel-case, ditto for the interface name,  but this is snake-case so I am not sure what you are trying to say here.